Project.toml

```diff
@@ -1,31 +1,15 @@
 name = "AbstractDifferentiation"
 uuid = "c29ec348-61ec-40c8-8164-b8c60e9d9f3d"
 authors = ["Mohamed Tarek <mohamed82008@gmail.com> and contributors"]
-version = "0.5.2"
+version = "0.6.0"
```

Comment:
There's nothing breaking in this PR, I think?
```suggestion
version = "0.5.2"
```

Reply:
rebased on master, comment no longer applies